modules/index/src/test/java/org/apache/ignite/internal/index/IndexManagerTest.java

```diff
@@ -226,7 +226,10 @@ private void createAndStartComponents() {
                 lowWatermark
         );
 
-        assertThat(allOf(metaStorageManager.start(), catalogManager.start(), indexManager.start()), willCompleteSuccessfully());
+        assertThat(
+                allOf(metaStorageManager.startAsync(), catalogManager.startAsync(), indexManager.startAsync()),
```

Comment:
Here and there, should we use startAsync() that you've introduced in such places?